src/util/FidoUtil.js

```diff
@@ -0,0 +1,56 @@
+/*!
+ * Copyright (c) 2015-2016, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+/* eslint complexity:[2, 9] */
+
+define(function () {
+  var fn = {};
+
+  fn.getU2fEnrollErrorMessageKeyByCode = function (errorCode) {
+    switch (errorCode) {
+      default:
+      case 1:
+        return 'u2f.error.other';
+      case 2:
+      case 3:
+        return 'u2f.error.badRequest';
+      case 4:
+        return 'u2f.error.unsupported';
+      case 5:
+        return 'u2f.error.timeout';
+    }
+  };
+
+  fn.getU2fVerifyErrorMessageKeyByCode = function (errorCode, isOneFactor) {
```

Comment:
and in the same line of my other comment, this can be
```
fn.getU2fVerifyErrorMessage = function (errorCode, isOneFactor) {
  ...
  return Okta.loc(key, 'login');
}
```